src/main/webapp/WEB-INF/templates/globalTemplate.xhtml

```diff
@@ -23,8 +23,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
 
-    <h:outputStylesheet library="webjars" name="bootstrap/3.3.2/css/bootstrap.min-jsf.css"/>
-    <h:outputStylesheet library="webjars" name="bootstrap/3.3.2/css/bootstrap-theme.min-jsf.css"/>
+    <h:outputStylesheet library="webjars" name="bootstrap/3.3.7-1/css/bootstrap.min-jsf.css"/>
```

Comment:
Why is the bootstrap dependency not updated in the POM in this PR? I mean the POM has the correct version, but it should be updated in this PR and not in some other PR.

Reply:
I think it was a problem with IntelliJ. IntelliJ proposed me another version for those libraries in the previous PR. Executing the connector, I didn't see any frontend problem. But doing some changes in the current PR, IntelliJ showed me the problem.